src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -305,11 +305,11 @@ static ImmutableArray<FunctionPointerParameterSymbol> makeParametersFromMetadata
 
             static RefKind getRefKind(ParamInfo<TypeSymbol> param, ImmutableArray<CustomModifier> paramRefCustomMods, RefKind hasInRefKind)
             {
-                // PROTOTYPE(func-ptr): Need to encode out params as a custom modifier of some kind
                 return param.IsByRef switch
                 {
                     false => RefKind.None,
                     true when CustomModifierUtils.HasInAttributeModifier(paramRefCustomMods) => hasInRefKind,
```

Comment:
>true when CustomModifierUtils.HasInAttributeModifier(paramRefCustomMods) => hasInRefKind, [](start = 20, length = 89)

How do we deal with a situation when there are both modifiers? Please add a test for a situation like that.

Reply:
This is a good test, will add.

---
In reply to: [409787440](https://github.com/dotnet/roslyn/pull/43355#discussion_r409787440) [](ancestors = 409787440)